src/modules/esl-share/core/esl-share.ts

```diff
@@ -38,6 +38,9 @@ export class ESLShare extends ESLBaseElement {
   /** Event to dispatch on ready state of {@link ESLShare} */
   @prop('esl:share:ready') public SHARE_READY_EVENT: string;
 
+  /** Event to dispatch on update of {@link ESLShare} */
+  @prop('esl:share:update') public SHARE_CHANGED_EVENT: string;
```

Comment:
```suggestion
  @prop('esl:share:update') public SHARE_UPDATE_EVENT: string;
```

Reply:
Event firing has been reworked on share components. Just only one event in case the config of the component is changed.